hack/tidy-check.sh

```diff
@@ -64,14 +70,14 @@ function tidy {
   mv "$SUM_FILE" "$TMP_SUM_FILE"
 
   if [ -n "$GO" ] && [[ "$($GO version|awk '{print $3}')" == $TARGET_GO_VERSION_PATTERN ]]; then
-    /usr/bin/env bash -c "$GO mod tidy" >>/dev/null 2>&1
+    /usr/bin/env bash -c "cd $PROJECT_DIR && $GO mod tidy" >>/dev/null 2>&1
```

Comment:
Should it use $DOCKER_TIDY_COMMAND too? Of course the variable should be renamed to "TIDY_COMMAND"

Reply:
I use this variable to distinguish the different commands between local dev environment and container environment. Because our project locates in different paths in these two environment, we need to change the command slightly. 

- For local dev environment, for both root path or plugin path, the command could be "cd $PROJECT_DIR && go mod tidy >> /dev/null 2>&1"
- For container environment, since the work directory is /usr/src/github.com/vmware-tanzu/antrea, we need to run either "go mod tidy >> /dev/null 2>&1" or "cd $1 && go mod tidy >> /dev/null 2>&1"

Reply:
I think $1 is relative path, so it should work for all cases? For example:
```
PROJECT_RELATIVE_DIR=${1:-.}
PROJECT_DIR=$(dirname "$THIS_DIR")/$PROJECT_RELATIVE_DIR
TIDY_COMMAND="cd $PROJECT_RELATIVE_DIR && go mod tidy >> /dev/null 2>&1"
```

Reply:
Thanks for the suggestion. It can help simplify the code here. But since the script is under hack folder, for local development environment, I think we need to use "cd .. && $TIDY_COMMAND".
Will update this part.

Reply:
Have updated to pushd antrea directory so that we can use the same $TIDY_COMMAND. Please help review the latest change, thanks.